spec/spec_helper.rb

```diff
@@ -1,8 +1,62 @@
-require 'coveralls'
-require 'simplecov'
+ENV['RACK_ENV'] = 'test'
```

Comment:
You can not remove `require 'coveralls'` and `simplecov`. If you do, the functionality of these services will be broaken, or am I missing something?